2216/2/versus.rb

```diff
@@ -0,0 +1,91 @@
+require 'optparse'
+require 'russian_obscenity'
+require 'unicode'
+require_relative 'badwordsanalizator'
+require_relative 'topwordsanalizator'
+require_relative 'battle'
+require_relative 'table_printer'
+require_relative 'rapper'
+
+# get env variables
+# --------------------------------------------------------------------------------------------
```

Comment:
I see some kind of business logic in this file.
Please, think about a class which will take the responsibility for all of this stuff.